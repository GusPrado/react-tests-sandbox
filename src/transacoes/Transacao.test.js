import React from "react";
import { render } from "@testing-library/react";

import Transacao from "./Transacao";

describe("Componente de transação do extrato", () => {
  it("o snapshot deste componente deve permanecer inalterado", () => {
    const { container } = render(
      <Transacao data="15/03/2021" tipo="saque" valor="200.00" />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
