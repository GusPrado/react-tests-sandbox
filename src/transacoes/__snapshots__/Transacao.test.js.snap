// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato o snapshot deste componente deve permanecer inalterado 1`] = `
<div
  class="transacao-container"
>
  <p>
    15/03/2021
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    200.00
  </p>
</div>
`;
